ivo-projects/todo/src/components/DoneButtons.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import {
+    FaSquareO,
+    FaCheckSquareO 
+} from 'react-icons/lib/fa';
+
+export const DoneButtons = (props) => {
+    if(props.isDone) {
+        return(
+            <button 
```

Comment:
You should not have two buttons but a single one. It should read the 'isDone' property from the reminder and be checked or unchecked accordingly. 

You should dispatch doneReminder with !isDone